tests/integration/helpers.py

```diff
@@ -75,6 +75,25 @@ def dump(self) -> Dict[str, Any]:
         return result
 
 
+async def destroy_cluster(ops_test: OpsTest, applications: list[str]) -> None:
+    """Destroy cluster in a forceful way."""
+    for app in applications:
+        await ops_test.model.applications[app].destroy(
+            destroy_storage=True, force=True, no_wait=False
```

Comment:
Does it break with `no_wait=True`? I think it might be nice to add so that the process speeds up, especially since we don't care about it being done safely. [Docs link](https://pythonlibjuju.readthedocs.io/en/latest/api/juju.application.html#juju.application.Application.destroy)

Reply:
I'll give it a try locally